app/models/api_user.rb

```diff
@@ -4,14 +4,15 @@
 #
 # To obtain an API token for a user, run this in the console:
 #
-#   DataAPIUser.tad_user.create_magic_link_token!
-class DataAPIUser < ActiveHash::Base
+#   APIUser.tad_user.create_magic_link_token!
+class APIUser < ActiveHash::Base
   include ActiveHash::Associations
   include AuthenticatedUsingMagicLinks
 
   self.data = [
-    { id: 1, name: 'User for testing, not used in production' },
-    { id: 2, name: 'DfE TAD' },
+    { id: 1, name: 'User for testing, not used in production', api_type: 'TestAPI' },
```

Comment:
- `api_type` is odd because it implies there's more than one API under the type. Maybe just `api`? Or `authorized_api` or equivalent?
- do we need a testing user for each API? Maybe we should give them high IDs starting at 100 or sth?

Reply:
> api_type is odd because it implies there's more than one API under the type. Maybe just api? Or authorized_api or equivalent?

I renamed to `authorized_api` for better clarity 👍 

> do we need a testing user for each API? Maybe we should give them high IDs starting at 100 or sth?

I don't think we do right now, the test user in our specs now is just to ensure that access is not given across different authorised APIs. Since there is only one now I think it's ok to leave with the current id. Let me know your thoughts